main.go

```diff
@@ -320,10 +333,15 @@ func cleanupMutes(s *discordgo.Session) {
 		err = s.GuildMemberRoleRemove(m.GuildId, m.User, env.RoleMute)
 		if err != nil {
 			log.Printf("Failed to remove role %s\n", err)
-			s.ChannelMessageSend(env.ChannelModlog, fmt.Sprintf("Failed to remove role Mute from user <@%s>. Error: %s", m.User, err))
+			if _, nestedErr := s.ChannelMessageSend(env.ChannelModlog, fmt.Sprintf("Failed to remove role Mute from user <@%s>. Error: %s", m.User, err)); nestedErr != nil {
+				log.Println("Failed to send Mute role removal message: " + err.Error())
+				return
```

Comment:
```suggestion
```

Reply:
Ah my bad, you're right; this isn't a fatal error. Changing this along with buffet's suggestions.